tests/parser/test_recipe_parser.py

```diff
@@ -19,9 +20,11 @@
 TEST_FILES_PATH: Final[str] = "tests/test_aux_files"
 
 # Long multi-line description string found in the `simple-recipe.yaml` test file
-SIMPLE_DESCRIPTION: Final[
-    str
-] = "This is a PEP '561 type stub package for the toml package.\nIt can be used by type-checking tools like mypy, pyright,\npytype, PyCharm, etc. to check code that uses toml."  # pylint: disable=line-too-long
+SIMPLE_DESCRIPTION: Final[str] = (
+    "This is a PEP '561 type stub package for the toml package."
+    "\nIt can be used by type-checking tools like mypy, pyright,"
+    "\npytype, PyCharm, etc. to check code that uses toml."
```

Comment:
FYI in other projects we've preferred to leave the newlines at the end

```suggestion
    "This is a PEP '561 type stub package for the toml package.\n"
    "It can be used by type-checking tools like mypy, pyright,\n"
    "pytype, PyCharm, etc. to check code that uses toml."
```